Model/Data/Product/LocalizedDataParser.php

```diff
@@ -63,6 +63,7 @@ public function parse(array $data, int $storeViewId, string $languageCode): arra
                 $datum['Id'] :
                 $datum['TextFieldsModel']['Sku'];
             $itemData['store_view_code'] = $storeViewCode;
+            $itemData['_store'] = $storeViewId;
```

Comment:
Why are we setting this?

Reply:
\Magento\CatalogImportExport\Model\Import\Product::getRowScope for reference, this key is required so that import recognises scope during import

\Magento\CatalogUrlRewrite\Observer\AfterImportDataObserver::isNeedToPopulateForUrlGeneration relevant for this function